Review.java

```diff
@@ -187,4 +187,30 @@ public static double totalSentiment(String fileName)
      }
      return sumOfSentiment;
   }
+  

```

Comment:
Maybe add comments explaining why you chose these numbers for the ratings. Also I agree with using greater than or equal too instead of greater than